integration-tests/src/test/java/tachyon/client/FileInStreamIntegrationTest.java

```diff
@@ -226,24 +228,23 @@ public void seekExceptionTest1() throws IOException {
    * Test <code>void seek(long pos)</code>. Validate the expected exception for seeking a position
    * that is past EOF.
    *
-   * @throws IOException
+   * @throws IllegalArgumentException
```

Comment:
updated exception type. Not sure what the convention is for documenting exceptions in tests because if the exception is thrown the test fails (not like other methods where a caller can catch and handle it).